controllers/druid/handler.go

```diff
@@ -391,6 +395,158 @@ func checkCrashStatus(sdk client.Client, m *v1alpha1.Druid) {
 	}
 }
 
+// scalePVCForSts shall scale validate pvc scaling, delete sts using orphan and patch pvc.
+func scalePVCForSts(sdk client.Client, drd *v1alpha1.Druid, ls map[string]string, configMapSHA, serviceName, nodeSpecUniqueStr string, nodeSpec *v1alpha1.DruidNodeSpec) {
+
+	sts := makeStatefulSetEmptyObj()
+	// During creation/initalization of druid cluster, this will fail since the sts will not be present
+	// So logging this as info, event type normal, subsequent reconciles should work fine.
+	if err := sdk.Get(context.TODO(), *namespacedName(nodeSpecUniqueStr, drd.Namespace), sts); err != nil {
+		msg := fmt.Sprintf("failed to get [StatefuleSet:%s] due to [%s]", nodeSpecUniqueStr, err.Error())
+		logger.Info(msg, "name", drd.Name, "namespace", drd.Namespace)
+		sendEvent(sdk, drd, v1.EventTypeNormal, "GET_FAIL", err.Error())
```

Comment:
so when a brand new druid cluster is created, we will create a whole bunch of GET_FAIL events ? I think, from the `err` we can check if it is caused by non-existence of sts and in that case we shouldn't log/event anything.